.changeset/nice-snakes-camp.md

```diff
@@ -0,0 +1,46 @@
+---
+"@salt-ds/core": minor
+---
+
+**_Theming and CSS updates_** with visual changes to Salt components:
+
+1. Minor changes in `Button` component width. `CTA` and `Primary` button text font weight changed from bold to semibold. Expect components containing Salt `Button` to have similar change.
```

Comment:
Is the change in width due to the font weight change or something else?

Reply:
Yes it's due to font weight

Reply:
I think we should update the wording so it's clearer that's the case.